BUILDGUIDE.md

```diff
@@ -148,6 +148,48 @@ Unix (`netcoreapp`):
+* Database "NORTHWIND" present in SQL Server, created using SQL script [createNorthwindDb.sql](tools/testsql/createNorthwindDb.sql). To setup an Azure Database with "NORTHWIND" tables, use SQL Script: [createNorthwindAzureDb.sql](tools/testsql/createNorthwindAzureDb.sql).
+* Make a copy of the configuration file [config.default.json](src/Microsoft.Data.SqlClient/tests/tools/Microsoft.Data.SqlClient.TestUtilities/config.default.json) and rename it to `config.json`. Update the values in `config.json`:
+
+|Property|Description|Value|
+|------|--------|-------------------|
+|TCPConnectionString | Connection String for a TCP enabled SQL Server instance. | `Server={servername};Database={Database_Name};Trusted_Connection=True;` <br/> OR `Data Source={servername};Initial Catalog={Database_Name};Integrated Security=True;`|
+|NPConnectionString | Connection String for a Named Pipes enabled SQL Server instance.| `Server=\\{servername}\pipe\sql\query;Database={Database_Name};Trusted_Connection=True;` <br/> OR <br/> `Data Source=np:{servername};Initial Catalog={Database_Name};Integrated Security=True;`|
+|TCPConnectionStringHGSVBS | (Optional) Connection String for a TCP enabled SQL Server with Host Guardian Service (HGS) attestation protocol configuration. | `Server=tcp:{servername}; Database={Database_Name}; UID={UID}; PWD={PWD}; Attestation Protocol = HGS; Enclave Attestation Url = {AttestationURL};`|
+|UseManagedSNIOnWindows | (Optional) Enables testing with Managed SNI on Windows| `true` OR `false`|
+
+Commands to run tests:
+
+Windows (`netfx x86`):
+```bash
+> dotnet test "src\Microsoft.Data.SqlClient\tests\PerformanceTests\Microsoft.Data.SqlClient.Performance.Tests.csproj" /p:Platform="Win32" /p:Configuration="Release" /p:TestTargetOS="Windowsnetfx" --no-build -v n --filter "category!=nonnetfxtests&category!=failing&category!=nonwindowstests"
+```
+
+Windows (`netfx x64`):
+```bash
+> dotnet test "src\Microsoft.Data.SqlClient\tests\PerformanceTests\Microsoft.Data.SqlClient.Performance.Tests.csproj" /p:Platform="x64" /p:Configuration="Release" /p:TestTargetOS="Windowsnetfx" --no-build -v n --filter "category!=nonnetfxtests&category!=failing&category!=nonwindowstests"
+```
+
+Windows (`netcoreapp`):
+```bash
+> dotnet test "src\Microsoft.Data.SqlClient\tests\PerformanceTests\Microsoft.Data.SqlClient.Performance.Tests.csproj" /p:Platform="AnyCPU" /p:Configuration="Release" /p:TestTargetOS="Windowsnetcoreapp" --no-build -v n --filter "category!=nonnetcoreapptests&category!=failing&category!=nonwindowstests"
+```
+
+Unix (`netcoreapp`):
+```bash
+> dotnet test "src/Microsoft.Data.SqlClient/tests/PerformanceTests/Microsoft.Data.SqlClient.Performance.Tests.csproj" /p:Platform="AnyCPU" /p:Configuration="Release" /p:TestTargetOS="Unixnetcoreapp" --no-build -v n --filter "category!=nonnetcoreapptests&category!=failing&category!=nonlinuxtests&category!=nonuaptests"
+```
+
+## Run A Single Test
```

Comment:
You could also include a section on how to collect benchmarks.

Reply:
Good idea. But don't you think it'd be better to add a section in ADO.NET documents for this purpose? Then we can put its link here.

Reply:
It's our test suite.. ADO.NET docs are for driver documentations.. so no.. we document things only here for test lab building.